src/search.cpp

```diff
@@ -185,11 +185,11 @@ void Search::init() {
 
```

Comment:
While we're at it, can you change this comment

    /// Search::reset() clears all search memory, to restore a deterministic state

to:

    /// Search::reset() clears all search memory, to obtain reproducible search results

The word "deterministic" was a poor choice on my part. It means the relationship cause to consequence is a constant. Here it's different, we reset the cause to the same state, to guarantee the same consequence, so "reproducible" is a better word.